### ISSUE TITLE
Nvy with new neovim does not start when a filename passed in command line

### ISSUE THREAD

**Issue by VladimirMarkelov** [RMichelsen/Nvy#53]
I updated neovim to `0.7.2` version that has been issued recently. Now if I run `nvy`, it starts successfully with empty workspace and I can `e: file.txt` and edit the file. But when I start `nvy` with a filename to edit, e.g. `nvy file.txt`, it opens the application, hangs for a few seconds without drawing any text in text area and then `nvy` crashes. No messages, I found only this in Windows event log:

```
Faulting application name: Nvy.exe, version: 0.0.0.0, time stamp: 0x61e9d774
Faulting module name: Nvy.exe, version: 0.0.0.0, time stamp: 0x61e9d774
Exception code: 0xc000041d
Fault offset: 0x0000000000004faa
```

I tried previous `nvy` versions. They crash in the same way.

**Comment by statiolake** [comment]
I'm not sure, but I had a similar problem, which I fixed in #50.

**Comment by VladimirMarkelov** [comment]
Version `0.2.6` works great. The issue is fixed.
